plugins/serverless/src/tasks/run.ts

```diff
@@ -0,0 +1,62 @@
+import { Task } from '@dotcom-tool-kit/types'
+import { ServerlessOptions, ServerlessSchema } from '@dotcom-tool-kit/types/lib/schema/serverless'
+import { spawn } from 'child_process'
+import { VaultEnvVars } from '@dotcom-tool-kit/vault'
+import { hookConsole, hookFork } from '@dotcom-tool-kit/logger'
+import getPort from 'get-port'
+import waitPort from 'wait-port'
+
+export default class ServerlessRun extends Task<typeof ServerlessSchema> {
+  static description = ''
```

Comment:
**nitpick**: could you add a short description for the task here? we should get into the habit of doing this, it will enable us to display the tasks and hooks to users in a friendlier way in the future.

Reply:
Oh yes good point!

Reply:
Done :)